docs/samples/explanation/aix/Fetch_20newsgroups/README.md

```diff
@@ -0,0 +1,40 @@
+# Using AIX to get explanations for NewsGroups classifications
```

Comment:
Let's only leave the code examples here in kserve repo and move the documentation to website [explainer folder](https://github.com/kserve/website/tree/main/docs/modelserving/explainer/aix)

Reply:
Ok we'll update the docs to there